doc/fping.pod

```diff
@@ -155,6 +155,12 @@ to any target (default is 10, minimum is 1).
 
 Set the interface (requires SO_BINDTODEVICE support).
 
+=item B<--icmp-timestamp>
+
+Send ICMP timestamp requests (ICMP type 13).
```

Comment:
Perhaps add the following to the end of the sentence?

` instead of ICMP Echo requests`